Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1233/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #373 of commit dd3191dee39ca9a818893173b48fdd863351b5dc automatically merged.
GitHub pull request #373 of commit dd3191dee39ca9a818893173b48fdd863351b5dc automatically merged.
GitHub pull request #373 of commit dd3191dee39ca9a818893173b48fdd863351b5dc automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git6563296872287975339.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
No valid HEAD. Skipping the resetting
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git647602209396024546.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/373/merge^{commit} # timeout=10
 > git branch -a --contains 8a7e1f3a9db3e1219dd2267e2b7ddd2e3e4c164a # timeout=10
 > git rev-parse remotes/origin/pr/373/merge^{commit} # timeout=10
Checking out Revision 8a7e1f3a9db3e1219dd2267e2b7ddd2e3e4c164a (origin/pr/373/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 8a7e1f3a9db3e1219dd2267e2b7ddd2e3e4c164a
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson2443253924196557805.sh
+ cd /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ git pull
Already up-to-date.
[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/deploy_code.yml -i /tmp/inventory6714530915204907278.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1233 -e server_docroot_folder=/var/www -e deploy_type=local -e profile_source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode -e source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/docroot -e profile_destination=profiles/contrib/openy -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Generate commentinfo.md file] ************ 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Make sure folder is not exist] *********** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Sync files into docroot] ***************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Sync OpenY profile] ********************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Folder permissions] ********************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=6    changed=4    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/build.yml -i /tmp/inventory2745156494841050770.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1233 -e server_docroot_folder=/var/www -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Folder permissions] **************************************************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/www/build1233/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal1233 -e drupal_folder=/var/www/build1233 -e site_url=http://openy-dev.ffwua.com/build1233 -e pp_environment=default -e run_reinstall=true -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
failed: [localhost] => {"failed": true, "parsed": false}
BECOME-SUCCESS-ogllmnfqoqbkgeesbgsrbkgjcrzpuezf
Traceback (most recent call last):
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 2012, in <module>
    main()
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 291, in main
    changed |= recursive_set_attributes(module, file_args['path'], follow, file_args)
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 132, in recursive_set_attributes
    changed |= module.set_fs_attributes_if_different(tmp_file_args, changed)
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 1207, in set_fs_attributes_if_different
    file_args['path'], file_args['mode'], changed
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 1045, in set_mode_if_different
    path_stat = os.lstat(path)
OSError: [Errno 2] No such file or directory: '/tmp/git3262018958368964875.credentials'


FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=3    changed=0    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

--- a/openy.install
+++ b/openy.install
@@ -269,7 +269,7 @@ function openy_update_8020() {
 }
 
 /**
- * Fix typo in theme settings.
+ * Fix typos in theme settings.
  */
 function openy_update_8021() {
   // Get config data.

